src/core/toxid.h

```diff
@@ -41,9 +41,9 @@ class ToxId
 public:
     ToxId(); ///< The default constructor. Creates an empty Tox ID.
     ToxId(const ToxId& other); ///< The copy constructor.
-    ToxId(const QString& id); ///< Create a Tox ID from QString.
-                              /// If the given id is not a valid Tox ID, then:
-                              /// publicKey == id and noSpam == "" == checkSum.
+    explicit ToxId(const QString& id); ///< Create a Tox ID from QString.
+                                       /// If the given id is not a valid Tox ID, then:
+                                       /// publicKey == id and noSpam == "" == checkSum.
```

Comment:
>What about commets to signals?

Fits nicely with a "top" block. Let's say there's a class with several signals and an enum. Would become:

```c++
/*…license info…*/
#include "blabla.h"

static void method() 
{ 
    // I'm just a little example.
}

/**
@class OurClass
@brief Exists for some reason…!?

Longer description
*/

/**
@enum OurClass::OurEnum
@brief The brief description line.

@var EnumValue1
means something

@var EnumValue2
means something else

Optional long description
*/

/**
@fn OurClass::somethingHappened(const QString &happened)
@param[in] happened    tells what has happened…
@brief This signal is emitted when something has happened in the class.

Here's an optional longer description of what the signal additionally does.
*/